core/src/main/resources/error/error-classes.json

```diff
@@ -1191,6 +1191,12 @@
     ],
     "sqlState" : "42805"
   },
+  "OUTPUT_PATH_ALREADY_EXISTS" : {
```

Comment:
Does the `OUTPUT` word bring any additional info? Can we name the error class as `PATH_ALREADY_EXISTS`?

Reply:
+1